connecting-to-xdai/run-own-node.md

```diff
@@ -0,0 +1,116 @@
+
+### 1. Setting up a Digital Ocean Instance
+1. Buy a droplet (cloud computer). I recommend a machine with *at least* the following specs ($40/month):
+    * `Ubuntu 20.4 / 8 GB / 4 CPUs 160 GB SSD Disk 5 TB transfer`
+    * If this too slow, I synced a node in ~6 hours with the $80/month option
+      > If your sync is very slow (extended beyond two days) then very likely your setup cannot catch up with the chain progress.
+    * ![](../.gitbook/assets/xdai-node/droplet.png)
+
+2. Remotely [connect](https://docs.digitalocean.com/products/droplets/how-to/connect-with-ssh/) to the Droplet using ssh.
+* You should see the following on success:
+* ![](../.gitbook/assets/xdai-node/login.png)
+
+### 2. Install Nethermind and dependencies
+
+*Note: these commands are verbatim from [here](https://docs.nethermind.io/nethermind/ethereum-client/running-nethermind/running-the-client)*
+
+
+1. `wget [NETHERMIND_URL]` - use amd64 release from [downloads](https://downloads.nethermind.io/))
+2. ```sudo apt-get update && sudo apt-get install libsnappy-dev libc6-dev libc6 unzip -y``` - Linux dependencies
+3. `unzip NETHERMIND_URL -d nethermind` - unzip to nethermind folder
+4. `cd nethermimd`
+
+At this point, I diverged from the basic installation. This is because there are some specific configuration settings that are nice for running a node (and playing Dark Forest). 
+
+### 3. Configure Nethermind for Dark Forest
+
+So in the `nethermind` folder you will see `Nethermind.Launcher` and `Nethermind.Runner`.
+
+* ![](../.gitbook/assets/xdai-node/nethermind.png)
+
+From [ethhub](https://docs.ethhub.io/using-ethereum/ethereum-clients/nethermind/)
+
+> Nethermind.Launcher is an interactive prompt that allows you to choose the network which then calls Nethermind.Runner with the appropriate config file. This is the quickest way to get up and running.
+> Nethermind.Runner is the actual client binary similar to the geth binary and supports both cli options or a JSON configuration file. `Nethermind.Runner --config <path_to_config>`
+
+I added some custom configurations, so I choose to use the `Nethermind.Runner` binary with my custom config file.
+
+1. `cd nethermind/configs` 
+  * We want to sync an xDai node, so here are the options for xDai (xdai_dfdao is my custom config): 
+  * ![](../.gitbook/assets/xdai-node/config.png)
+  * I choose to copy `xdai_pruned_mev.cfg` because:
+    1. I want the experiment with [mev](https://www.xdaichain.com/for-developers/developer-resources/mev-and-flashbots) on xDai
+    2. I want to save size by pruning, at the cost of historical queries:  
+        > pruning is the process of removing some of the intermediary state nodes - it saves some disk space but makes most of the historical state queries fail.
+
+2. `cp CONFIG_FILE YOUR_CONFIGNAME`
+  * for me, this was `cp xdai_pruned_mev.cfg xdai_dfdao.cfg`
+
+Edit your config. Here is [mine](https://gist.github.com/cha0sg0d/0c633efcb11db3b869a4450fb6035d7d) for reference.
```

Comment:
Lets not link to external crypto related gists as they can change out under the hood
Going to have to explain vim or better yet nano as it says how to quit on the ui..
```suggestion
Edit your config file with something like `nano configs/xdai_dfdao.cfg` and change the following fields to allow access to your node from the internet:
```